pkg/server/server.go

```diff
@@ -0,0 +1,36 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/pullsheet/pkg/client"
+	"github.com/google/pullsheet/pkg/server/job"
+	"net/http"
+)
+
+type Server struct {
+	ctx context.Context
```

Comment:
context should never be embedded into a struct : https://blog.golang.org/context-and-structs

Always explicitly pass `ctx` between functions as the first argument. For HTTP, there is a handy `r.Context()` function you can use to generate the right context for a request.